.github/workflows/release_to_pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish geopandas to PyPI
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-n-publish:
+    name: Build and publish geopandas to PyPI
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Build a binary wheel and a source tarball
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+          python setup.py sdist
+          python setup.py bdist_wheel --universal
```

Comment:
`--universal` is included in the current steps in the wiki

Reply:
Ah, universal actually means python 2 and 3. While the default is still pure python (cross version), but only python 3, I think